scipy/stats/_continuous_distns.py

```diff
@@ -8393,6 +8393,13 @@ class gennorm_gen(rv_continuous):
     .. [1] "Generalized normal distribution, Version 1",
            https://en.wikipedia.org/wiki/Generalized_normal_distribution#Version_1
 
+    .. [2] Nardon, Martina, and Paolo Pianca. "Simulation techniques for
+           generalized Gaussian densities." Journal of Statistical
+           Computation and Simulation 79.11 (2009): 1317-1329
```

Comment:
I see that our beta is their alpha, and the PDFs match except for the constant A:

![image](https://user-images.githubusercontent.com/6570539/149584455-e86c6739-0bc1-457c-b0a5-68277b92f258.png)

Does this affect anything about the random variate generation?

Reply:
> When the scale is 1, A=1?

Yes.

> Does this affect anything about the random variate generation?

Not really, the user can adjust the scale parameter to generate samples from the scaled distribution: something like `gennorm(beta=0.25, scale=2, size=100)` (here, samples are generated from the distribution where `A(alpha, sigma) = 2`).

> Can you explain how all this works out?

The way SciPy parameterizes the `gennorm` distribution `A(alpha) = 1`. So, the equation below:

![image](https://user-images.githubusercontent.com/43181252/149610925-e333afa9-5b70-467f-a7bc-ac126c193251.png)

simplifies to `Gamma(a = 1/beta, b = 1)` (in SciPy and NumPy, `scale` is the `b` parameter which is 1 by default). So, I just sample from `random_state.gamma(1/beta)`. Everything else is done as you explain in https://github.com/scipy/scipy/pull/15413#discussion_r785172470

For a more detailed explanation, @chrisb83 is the go-to expert here :) Feel free to add to this or point out if something I said is wrong

Reply:
I guess the way to think of it is that we are always choosing the variance _so that_ A=1. The variance here is not the same as our scale. With `scale = 1` fixed at 1, we get a different variance depending on alpha (our `beta`).